hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
+| 62            | GPIO A                        | PD14                                          |
+| 63            | GPIO B                        | PD13                                          |
+| 64            | GPIO C                        | PD12                                          |
+| 65            | GND                           | GND                                           |
+| 66            | GPIO D                        | PD11                                          |
+| 67            | GPIO E                        | PD10                                          |
+| 68            | GPIO F                        | PD9                                           |
+| 69            | GPIO G                        | PD8                                           |
+| 70            | 5V                            | --------------------------------------------- |
+| 71            | Reserved                      | --------------------------------------------- |
+| 72            | 3.3V                          | 3.3V                                          |
+| 73            | I2C B SDA                     | --------------------------------------------- |
+| 74            | I2C B SCL                     | --------------------------------------------- |
+| 75            | UART C TX                     | --------------------------------------------- |
+| 76            | UART C RX                     | --------------------------------------------- |
+| 77            | UART D TX                     | --------------------------------------------- |
+| 78            | UART D RX                     | --------------------------------------------- |
+| 79            | GND                           | GND                                           |
+| 80            | Reserved                      | --------------------------------------------- |
+| 81            | Reserved                      | --------------------------------------------- |
+| 82            | Reserved                      | --------------------------------------------- |
+| 83            | Reserved                      | --------------------------------------------- |
+| 84            | Reserved                      | --------------------------------------------- |
+| 85            | Reserved                      | --------------------------------------------- |
+| 86            | 5V                            | --------------------------------------------- |
+| 87            | USB Device ID                 | --------------------------------------------- |
+| 88            | 3.3V                          | 3.3V                                          |
+| 89            | UART B TX                     | --------------------------------------------- |
+| 90            | UART B RX                     | --------------------------------------------- |
+| 91            | ADC A                         | PB8 \| ADC9                                   |
```

Comment:
Should use `,` to separate functions, not `|`, especially since `|` needs to be escaped.

Reply:
Traditionally, we used the forward slash on the schematics, but because this is a table, I think it looks better that there is the pipe ( | ) separating the functions because it looks more like a table since we cannot use a col span.